dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerTest.kt

```diff
@@ -56,22 +63,58 @@ class CompanyDataControllerTest {
 
     @Test
     fun `post a dummy company and check if that specific company can be queried by its name`() {
-        val testCompanyInformation = testDataProviderForEuTaxonomyDataForNonFinancials
-            .getCompanyInformationWithoutIdentifiers(1).first()
-        tokenHandler.obtainTokenForUserType(TokenHandler.UserType.Admin)
-        val postCompanyResponse = companyDataControllerApi.postCompany(testCompanyInformation)
+        val storedCompany = postFirstCompanyWithoutIdentifiers()
         tokenHandler.obtainTokenForUserType(TokenHandler.UserType.SomeUser)
         val getCompaniesByNameResponse = companyDataControllerApi.getCompanies(
-            searchString = testCompanyInformation.companyName,
+            searchString = storedCompany.companyInformation.companyName,
             onlyCompanyNames = true,
         )
-        val expectedCompany = StoredCompany(postCompanyResponse.companyId, testCompanyInformation, emptyList())
+        val expectedCompany = StoredCompany(storedCompany.companyId, storedCompany.companyInformation, emptyList())
         assertTrue(
             getCompaniesByNameResponse.contains(expectedCompany),
             "Dataland does not contain the posted company."
         )
     }
 
+    @Test
+    fun `post two dummy companies and check if the distinct endpoint returns all values`() {
+        tokenHandler.obtainTokenForUserType(TokenHandler.UserType.Admin)
+        val testCompanyInformation = testDataProviderForEuTaxonomyDataForNonFinancials
+            .getCompanyInformationWithoutIdentifiers(2)
+        testCompanyInformation.forEach {
+            companyDataControllerApi.postCompany(it)
+        }
+        val distinctValues = companyDataControllerApi.getAvailableCompanySearchFilters()
+        assertTrue(distinctValues.sectors?.containsAll(testCompanyInformation.map { it.sector }) ?: false)
+        assertTrue(distinctValues.countryCodes?.containsAll(testCompanyInformation.map { it.countryCode }) ?: false)
+    }
+
+    @Test
+    fun `post a dummy company and check if that specific company can be queried by its country code and sector`() {
+        val storedCompany = postFirstCompanyWithoutIdentifiers()
+        val getCompaniesByCountryCodeAndSector = companyDataControllerApi.getCompanies(
```

Comment:
Variable name should not have a command as input

Reply:
Added "Response" to show that this variable holds the response data of a HTTP request